backend/typescript/services/implementations/authService.ts

```diff
@@ -175,12 +169,11 @@ class AuthService implements IAuthService {
       const userRole = await this.userService.getUserRoleByAuthId(
         decodedIdToken.uid,
       );
+      // const firebaseUser = await firebaseAdmin
+      //   .auth()
+      //   .getUser(decodedIdToken.uid);
 
-      const firebaseUser = await firebaseAdmin
-        .auth()
-        .getUser(decodedIdToken.uid);
-
-      return firebaseUser.emailVerified && roles.has(userRole);
+      return /* firebaseUser.emailVerified && */ roles.has(userRole);
```

Comment:
haha we have to remember to uncomment this once all the user management stuff is done